test/release/local_release_page_list.txt

```diff
@@ -0,0 +1,8 @@
+extrusion-query.html
```

Comment:
Could you also add a note about this file in the README: https://github.com/mapbox/mapbox-gl-js/blob/03cbeb1425ccf24216c52a2a07ed2142c42e2feb/test/release/README.md